src/UI/Component/Listing/Factory.php

```diff
@@ -46,4 +46,18 @@ public function ordered(array $items);
 	 */
 	public function descriptive(array $items);
 
+	/**
+	 * ---
+	 * description:
+	 *   purpose: >
+	 *      A workflow presents a list of steps that the user should tackle
+	 *      in a defined order.
+	 *   composition: >
+	 *     The workflow has a title and a list of workflow steps.
```

Comment:
Add `ordered listing` as rival here. Main difference to me is, that the entries in the workflow relate to some task, which is not the case for the ordered list.